sdk/dotnet/Makefile

```diff
@@ -70,8 +70,9 @@ test_all:: dotnet_test
 dist::
 	go install -ldflags "-X github.com/pulumi/pulumi/sdk/v2/go/common/version.Version=${DOTNET_VERSION}" ${LANGHOST_PKG}
 
-brew:: dist
-	go install -ldflags "-X github.com/pulumi/pulumi/sdk/v2/go/common/version.Version=${DOTNET_VERSION}" ${LANGHOST_PKG}
```

Comment:
Not sure - but this used `DOTNET_VERSION` which was derived from `pulumictl get version --language dotnet`.  Does that do anything different than the vanilla `scripts/get-version HEAD`?

Reply:
Also - weird that this was building things twice previously (once in dist and once in brew)?

Reply:
DOTNET_VERSION keeps the *new* functionality - the pulumictl based one so we can continue to build the feature-3.0 alphas etc

The BREW_VERSION is a variable specifically using the get-version script - that way, homebrew is the only thing using that script

Yes, I did find that we were building the DotNet binary twice :D